tests/tests.js

```diff
@@ -249,23 +249,31 @@ describe('Get Zone letter from lat', function(){
     });
   });
 });
-describe('Parse USNG', function(){
-  describe('with single digit zone', function(){
-    it('should return zone=5; letter=Q', function(){
+describe('Parse USNG', function() {
+  describe('with single digit zone', function () {
+    it('should return zone=5; letter=Q', function () {
```

Comment:
These spacing changes are inconsistent. These should be made consistent with a formatter. If one is not currently set up in this repo, then it should be considered out of scope for this pull request and the spacing should remain unchanged for the time being.